storage/include/rocksdb/native_write_batch.ipp

```diff
@@ -55,6 +55,12 @@ void NativeWriteBatch::del(const std::string &cFamily, const KeySpan &key) {
   detail::throwOnError("batch del failed"sv, std::move(s));
 }
 
+template <>
+inline void NativeWriteBatch::del<::rocksdb::Slice>(const std::string &cFamily, const ::rocksdb::Slice &key) {
```

Comment:
Nit: Wouldn't `detail::toSlice()` work on a `::rocksdb::Slice` too? If yes, we can consider removing this specialization.